backends/audiocd/linux/linux-audiocd.cpp

```diff
@@ -332,6 +334,12 @@ bool LinuxAudioCDManager::play(int track, int numLoops, int startFrame, int dura
 	return true;
 }
 
+bool LinuxAudioCDManager::playAbsolute(int startFrame, int numLoops, int duration, bool onlyEmulate,
+		Audio::Mixer::SoundType soundType) {
+	// TODO
```

Comment:
I would put a `warning()` here, so devs know if suddenly this gets triggered.

But in general, will it work with emulation, like you did in the macOS backend?